common/djangoapps/xblock_django/models.py

```diff
@@ -40,3 +40,36 @@ def disabled_block_types(cls):
             return ()
 
         return config.disabled_blocks.split()
+
+
+class XBlockDeprecatedAdvancedComponentConfig(ConfigurationModel):
+    """
+    Configuration for deprecated XBlocks that should not be created in Studio.
+    """
+
+    class Meta(ConfigurationModel.Meta):
+        app_label = 'xblock_django'
+
+    disabled_blocks = TextField(
+        default='', blank=True,
+        help_text=_(
+            "Adding components in this list will disable the creation of new problem for "
+            "those components in Studio. Existing problems will work fine and one can edit"
```

Comment:
I think we should mention that list is space separated. 

@ehteshamkafeel Should we make this list comma separated? Or there is no chance that advance modules names will ever have space in them?